.codecov.yml

```diff
@@ -1,27 +1,46 @@
 codecov:
-  notify:
-    require_ci_to_pass: yes
+    notify:
+        require_ci_to_pass: yes
 
-coverage:
-  precision: 2
-  round: nearest
-  range: "50...100"
+ignore:
+    - '**/tests'
+    - '**/test'
+    - 'tools/**'
+    - 'packages/js/admin-e2e-tests'
+    - 'packages/js/api-core-tests'
+    - 'packages/js/create-woo-extension'
+    - 'packages/js/e2e-core-tests'
+    - 'packages/js/e2e-environment'
+    - 'packages/js/e2e-utils'
+    - 'packages/js/eslint-plugin'
+    - 'packages/js/internal-e2e-builds'
+    - 'packages/js/internal-js-tests'
+    - 'packages/js/internal-style-build'
+    - '**/*.test.*'
 
-  status:
-    project:
-      default:
-        informational: true
-    patch:
-      default:
-        informational: true
-    changes: off
+coverage:
+    precision: 2
+    round: nearest
+    range: '50...100'
```

Comment:
Looking at their docs, the first number defines at what point the graph is "red" and the second number determines at what coverage we reach "green". IMHO `80` is already an amazing coverage that might deserve being shown in green. ☺️